### PR TITLE
Code coverage harmonization

### PR DESCRIPTION
**Pull request by m-goggins**
- Up to 100% coverage for fhir and core harmonization scripts
- Added an additional test hl7 file with malformed/invalid segment_id fields
- Changed the import of copy to remove an error related to using copy.deepcopy() in the case of an invalid segment_id field

### CODE REVIEW

File: tests/harmonization/test_harmonization.py [hunk 1]
```diff
@@ -18,6 +18,11 @@ def test_standardize_hl7_datetimes():
     message_2 = open(
         pathlib.Path(__file__).parent.parent / "assets" / "FileSingleMessageLongTZ.hl7"
     ).read()
+    message_3 = open(
```
Comment (KennethSkylight) [inline tests/harmonization/test_harmonization.py:21]: could we make the message variable names more descriptive of what they're testing for like `message_invalid_segments`?

**Review by KennethSkylight — CHANGES_REQUESTED** [review]
One small thing

File: tests/harmonization/test_harmonization.py (hunk 1 above)
Comment (m-goggins) [reply tests/harmonization/test_harmonization.py:21]: Yeah, good point! I saw more descriptive names in another test file later but forgot to change these ones.

**Review by KennethSkylight — APPROVED** [review]
LGTM!

**Pull request merged**